### ISSUE TITLE
Add DBUS support for viewer

### ISSUE THREAD

**Issue by vpetersson** [Screenly/screenly-ose#559]
**tl;dr:** Let's introduce an IPC for server/viewer communication

Currently, the architecture for Screenly OSE looks something like this:

    Server -> Sqlite database <- Viewer

When a change is made to the playlist, the server writes the changes to the database, and then the viewer polls the database for changes. While this approach is simple, it prevents us from doing things like more reactive events over an API.

To remedy this, I'd like to introduce DBUS support in viewer as an IPC and introduce a signal for skipping to the next asset (like `sigusr1()`). We can then expand with more of these later.

**Comment by vpetersson** [comment]
Deprecated after introduction of new API.
